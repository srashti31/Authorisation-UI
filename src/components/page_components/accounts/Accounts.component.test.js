import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import renderer from "react-test-renderer";
import Accounts from "./Accounts.component";

configure({ adapter: new Adapter() });

describe("<Accounts />", () => {
    it("should match snapshot", () => {
        const accounts = {
            everydayAccount: "000-123 1123 3323",
            savingsAccount: "000-123 7738 7383",
        };
        const snapshot = renderer.create(<Accounts accounts={accounts} />).toJSON();
        expect(snapshot).toMatchSnapshot();
    });
    it("should render without crashing", () => {
        const accounts = {
            everydayAccount: "000-123 1123 3323",
            savingsAccount: "000-123 7738 7383",
        };
        shallow(<Accounts accounts={accounts} />);
    });
});
