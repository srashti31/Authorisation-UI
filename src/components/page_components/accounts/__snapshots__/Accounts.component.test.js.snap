// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Accounts /> should match snapshot 1`] = `
<div
  className="MuiPaper-root MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    aria-disabled={false}
    aria-expanded={false}
    className="MuiButtonBase-root MuiExpansionPanelSummary-root pannel-title"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex={0}
  >
    <div
      className="MuiExpansionPanelSummary-content"
    >
      Accounts
    </div>
    <div
      aria-disabled={false}
      aria-hidden={true}
      className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role={null}
      tabIndex={null}
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </span>
    </div>
  </div>
  <div
    className="MuiCollapse-container MuiCollapse-hidden"
    style={
      Object {
        "minHeight": "0px",
      }
    }
  >
    <div
      className="MuiCollapse-wrapper"
    >
      <div
        className="MuiCollapse-wrapperInner"
      >
        <div
          role="region"
        >
          <div
            className="MuiExpansionPanelDetails-root"
          >
            <div>
              <div
                className="info-title"
              >
                Everyday Accounts
              </div>
              <div
                className="description"
              >
                000-123 1123 3323
                 
              </div>
              <hr
                className="hr-detail"
              />
              <div
                className="info-title"
              >
                Savings Account
              </div>
              <div
                className="description"
              >
                000-123 7738 7383
                 
              </div>
              <hr
                className="hr-detail"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
